setup.cfg

```diff
@@ -1,9 +0,0 @@
-[flake8]
-ignore=E501
-
-[tool:pytest]
-python_files=test_*.py
-addopts=-n=auto --verbose -r=a --junit-xml=results/junit.xml
-testpaths=tests
-xfail_strict=true
-base_url=https://snippets.allizom.org
```

Comment:
We shouldn't remove this section entirely - did you mean to move it into `tox.ini`?